extensions/wikia/PortableInfoboxBuilder/PortableInfoboxBuilderController.class.php

```diff
@@ -37,20 +53,36 @@ private function attemptSave( $params ) {
 			$status->fatal( 'user-cant-edit' );
 		}
 
-		return $status->isGood() ? $this->save( $title, $params[ 'data' ] ) : $status;
+		//check if there is no write conflict
```

Comment:
there is a lot of things going on in this function. What do you thing about breaking it into smaller methods ?